### DISCUSSION TITLE
fa/manager/domain-worker-cdn-and-tunneling/How-to-setup-relay-server-using-Hiddify-Relay-Builder/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4092] [Content Help کمک به اصلاح محتوا]
# fa/manager/domain-worker-cdn-and-tunneling/How-to-setup-relay-server-using-Hiddify-Relay-Builder/

The proxy manager platform.

https://hiddify.com/fa/manager/domain-worker-cdn-and-tunneling/How-to-setup-relay-server-using-Hiddify-Relay-Builder/

**Comment by miladfff** [comment]
سلام خودتون کدوم روش رو پیشنهاد میکنید

**Reply by ReturnFI** [reply]
سلام.
تمام روش ها تست شده هستن اما در طول فیلترینگ روش dokodemo-door پایداری بیشتر داشت.

**Comment by ghamirizadi** [comment]
Ip table بعد از ریستارت 
Inactive میشه

**Reply by ReturnFI** [reply]
بعد از ریستارت rules هم پاک میشن؟

**Comment by reza-naghizadeh** [comment]
سلام من دوکودمو رو نصب کردم روی سرور میانی سرور اصلی هم کار میکنه ولی با ساب دامین سرور میانی نمیتونم به پنل وصل بشم و کانفینگ هاش هم کار نمیکنه

**Reply by ReturnFI** [reply]
سلام
سرور میانی ایران هست یا خارج؟

**Reply by reza-naghizadeh** [reply]
ایران

**Reply by reza-naghizadeh** [reply]
امکانش هست ssh هر دو سرور رو در اختیارتان قرار بدم تا پنل رو پیاده سازی کنین از آموزشها هرچقدر دنبال میکنم بجایی نمی‌رسم.

**Reply by ReturnFI** [reply]
اول چک کنید سرور ایران اکسس یا محدود نشده باشه 
سرور ایران IPv6 داره ؟
از روش های دیگه استفاده کردید؟
نه امکانش نیست.

**Reply by reza-naghizadeh** [reply]
بله سرور تمیزه همه روش ها رو زدم هیچکدوم انگار تانل نمیکنن البته یبار با ریورس زدم تانل شد اونم بعد چند ساعت دیگه کار نکرد

**Reply by ReturnFI** [reply]
از این [آموزش ](https://hiddify.com/fa/manager/domain-worker-cdn-and-tunneling/Tutorial-for-setting-up-local-IPv6/) و تانل Dokodemo-Door تست بگیرید

**Reply by ehsanps7** [reply]
سلام من اموزش ipv6 لوکالی که گذاشتینو انجام دادم اخرشم پینگ گرفتم سرور اصلی ای پی v6 سرور میانی رو پینگ میده وسرور میانی رو به منیجر(سرور اصلیی) اضافه کردم ولی کانفینگ ها کارنمیکنن
کجارو باید برسی کنم بنظرتون؟

**Reply by TomCat-Hub** [reply]
درود بر تو احسان عزیز،
وقت بخیر

اگر دقیقا مثل آموزش پیش رفته باشی و آیپی سرورها مشکلی نداشته باشه بطورکلی نباید ایرادی باشه و باید کانفیگ ها بخوبی کار کنن. برای فمیدن مشکل و ارایه راه حل نیاز به برسی دقیق تری هست. پس صمیمانه درخواست میکنم مشکلت رو با ذکر جزییات دقیق توی گروه تلگرام هیدیفای و تاپیک هیدیفای منیجر مطرح کنی تا دوستان برسی کنن و پیامت رو پاسخ بدن. 

لینک گروه تلگرام هیدیفای:
https://t.me/hiddify_board

بسیار متشکرم که هیدیفای رو انتخاب کردی
مانا باشی

**Comment by zekerya** [comment]
نصب با داکر و ریورس تانل هم اضافه کنید.

**Reply by ReturnFI** [reply]
سعی میکنم در آپدیت های بعدی این موارد اضافه کنم 
اما بخاطر مصرف بیش از اندازه منابع توسط تانل ریورس شاید این مورد اضافه نشه.

**Reply by rbsdotnet** [reply]
دردود وقت بخیر امکانش هست اطلاعات سرور ایران و خارج رو دراختیار بگذارم و پنل رو نصب کنید؟
من هرکاری نصب نمیشه
با انواع و اقسام خطاها روبه رو میشم
سپاس

**Reply by TomCat-Hub** [reply]
درود بر شما دوست عزیز
وقت بخیر
متاسفانه در حال حاضر این امکان فراهم نمیباشد.
لطفا جهت یافتن مشکل و پیدا کردن راه حل، مشکل خودتون رو با ذکر جزییات در **گروه تلگرامی هیدیفای** و تاپیک مربوط به **هیدیفای منیجر** مطرح کنید تا دوستان شما رو راهنمایی کنند.

لینک گروه تلگرامی هیدیفای:
https://t.me/hiddify_board

باتشکر از انتخاب شما
مانا باشید

**Comment by bazyari47** [comment]
باسلام
آیا امکان اتصال همزمان 2 نوع تانل وجود دارد؟
به عنوان نمونه هم گوست هم دوکودمو دور

**Reply by TomCat-Hub** [reply]
درود بر شما
وقتتون بخیر

خیر، متاسفانه امکانش فراهم نیست.
لطفا اگر سوال بیشتری داشتید و یا نیاز به راهنمایی بیشتری بود، توی گروه تلگرام هیدیفای و تاپیک مربوط به مشکلتون پیام بفرستید. این سریع ترین روشی هست که میتونید به درست ترین پاسخ برسید.

لینک گروه تلگرام هیدیفای:
https://t.me/hiddify_board

باتشکر از انتخاب شما
مانا باشید

**Comment by freeforlife-star** [comment]
سلام
من برنامه رو نصب کردم روی سرور ایران
بعد که اتصال اس اس اچ بسته میشه، نمیتونم به اون صفحه تانل ها متصل بشم و بایداسکریپت نصب رو دوباره بزنم
مثل خوده پنل هیدیفای نمیشه با ی دستور کوچیک دوباره پنل رو آورد و نخواد که دوباره نصب بشه ؟

**Reply by TomCat-Hub** [reply]
درود بر تو دوست عزیز
وقت بخیر

خیلی ممنونم که برای ما کامنت گذاشتید و سوال خودتون رو با ما درمیون گذاشتید.

متاسفانه در حال حاضر دستوری برای اجرای منو وجود ندارد و شما با همون دستور اسکریپت میتونید منو رو اجرا کنید و مشکلی پیش نخواهد آمد. با این وجود، نظر شما را با توسعه دهنده به اشتراک گذاشتیم و بزودی دستور و کامندی به منظور اجرای منو در نظر خواهند گرفت.

باتشکر از انتخاب شما
مانا باشید

**Reply by mhosamsh** [reply]
میتونی اول با wget دانلودش کنی بعد بهش اجازه اجرا شدن بدی با chmod +x و اگه خواستی برای راحتی کارت براش alias تعریف کنی.

**Comment by Raminraven** [comment]
سلام اگه سرور خارج فیلتر باشه کدوم تانل وصل میشه؟

**Comment by aghajani82** [comment]
سلام. در مورد (هاپروکسی) میشه توضیح بدین که از چه پروتکلی استفاده میشه. یا اگه مقاله سراغ دارید در این رابطه ممنون میشم معرفی کنید.

**Comment by alirezarah** [comment]
سلام کدوم روش رو رو سرور ایران نصب بشه بهتره؟

**Comment by alishefaee** [comment]
Consider my relay server VM machine only has a few ports mapped to the public network like 80=>8080. 
Then how relay server should listen on these ports from client?
(Suppose we already have static IP on the sever )
